messages/en.js

```diff
@@ -100,13 +100,13 @@ Blockly.Msg.OBJECT_KEYS_TOOLTIP = "Get all keys from the object";
 // Extra blocks
 // =========================================================================================
 Blockly.Msg.TEXT_SPECIAL_CHARACTER = "special character %1";
-Blockly.Msg.TEXT_SPECIAL_CHARACTER_TOOLTIP = "A special character that shouldn't be escaped (by a '\' prefix";
+Blockly.Msg.TEXT_SPECIAL_CHARACTER_TOOLTIP = "A special character that shouldn’t be escaped (by a '\\' prefix)";
 Blockly.Msg.TEXT_SPECIAL_CHARACTER_LF = "line feed (\\n)";
 Blockly.Msg.TEXT_SPECIAL_CHARACTER_CR = "carriage return (\\r)";
 Blockly.Msg.TEXT_SPECIAL_CHARACTER_CRLF = "carriage return line feed (\\r\\n)";
 Blockly.Msg.TEXT_SPECIAL_CHARACTER_TAB = "tab (\\t)";
 Blockly.Msg.TIMER_CONDITIONAL = "Repeat every %1 seconds, until %2 %3";
```

Comment:
For the text of a block (the operation of the block), should the text/sentence start with capital? Here is an example with capitalization, while most of the other block strings start with a lower-case. What is used with the blocks coming from the original Blockly if we want to match the style? Once the capitalization style is chosen, we could uniformize the translations, like in French.

Reply:
Seems that the standard blocks from Google indeed have a capital at the start of their tooltips:

![image](https://user-images.githubusercontent.com/14224149/130400904-12950537-f522-4237-be1f-762852eaa759.png)

If you have time, you can change that.  But I have to continue with other stuff on my backlog....

Reply:
I uniformized it in 34c153562da0813f8137e43720f3a91f94bdc85c